repository_service_tuf_api/common_models.py

```diff
@@ -86,6 +86,36 @@ class TUFSigned(BaseModel):
 
     class Config:
         fields = {"type": "_type"}
+        # https://docs.pydantic.dev/latest/api/config/#pydantic.config.ConfigDict.extra
+        model_config = ConfigDict(
```

Comment:
I am new to Pydantic and haven't seen this cnfig-as-nested-class pattern in the Pydantic docs. What I've seen is that model config is usually written into the `self.model_config` attribute inherited from `BaseModel`. Out of curiosity can you tell me more about the pattern you are using?

Reply:
I'm not sure that I can answer you why I use it that way, but at least it is consistent in all the models for RSTUF API 😆 

Once upon a time,  Kairo was looking at how to create a field alias for the TUF _type field, and he found https://docs.pydantic.dev/1.10/usage/model_config/, which has an example.

Reply:
Ah, thanks, I understand. This pydantic v1 syntax ... see https://docs.pydantic.dev/1.10/blog/pydantic-v2-alpha/#changes-to-config:

> To specify config on a model, it is now deprecated to create a class called Config in the namespace of the parent BaseModel subclass. Instead, you just need to set a class attribute called model_config to be a dict with the key/value pairs you want to be used as the config.





Maybe (as a low prio task) we could consider migrating to v2: https://docs.pydantic.dev/2.0/migration/

Reply:
Yes, I agree
- https://github.com/repository-service-tuf/repository-service-tuf-api/issues/532